smartapps/curb/curb-energy-monitor.src/curb-energy-monitor.groovy

```diff
@@ -150,96 +142,105 @@ private removeChildDevices(delete) {
 }
 
 def getCurbLocations() {
+    log.debug "Getting curb locations"
     def params = [
         uri: "http://app.energycurb.com",
         path: "/api/locations",
-        headers: ["Authorization": "Bearer ${atomicState.authToken}"]
+        headers: ["Authorization": "Bearer ${state.authToken}"]
     ]
-
+    def allLocations = [:]
     try {
         httpGet(params) {
             resp ->
                 def locationNameList = []
             def locationLookup = []
             resp.data.each {
+                log.debug "Found location: ${it}"
+                allLocations[it.id] = it.label
                 locationNameList.push(it.name)
                 locationLookup.push(it)
             }
-            atomicState.locationNames = locationNameList
-            atomicState.locationLookup = locationLookup
+            state.locationNames = locationNameList
+            state.locationLookup = locationLookup
         }
     } catch (e) {
         log.error "something went wrong: $e"
     }
+    log.debug "Found locations: ${allLocations}"
+    allLocations
```

Comment:
are we missing a return statement here?

Reply:
Also, `getCurbLocations` is not being called anywhere? The only place where it is being called, it is commented out right now?